src/main/scala/goatrodeo/envelopes/Envelope.scala

```diff
@@ -165,7 +165,7 @@ case class IndexFileEnvelope(
 object IndexFileEnvelope {
 
   def build(
-      version: Int = 1,
+      version: Int = 2,
```

Comment:
Under what circumstances will this change?

Reply:
If there's a breaking change in the underlying file format.

Over time, we should support backward compatibility with older formats, but given that nobody is using Big Tent/Goat Rodeo except us, we won't support backward compatibility right now.